cypress/integration/granules_spec.js

```diff
@@ -634,33 +628,20 @@ describe('Dashboard Granules Page', () => {
       cy.contains('button', 'Delete').click();
       cy.contains('.button--submit', 'Confirm').click();
 
-      cy.route2({
```

Comment:
This was never being hit before, and I verified that changing the property test to a different value that the tests still passed.  The new cy.intercept on L609-614, intercepts the request tests the body to have correct action, and then passes back a success now.